lib/controls/doc_field_text_edit.dart

```diff
@@ -46,6 +46,7 @@ class DocFieldTextEditState extends ConsumerState<DocFieldTextEdit> {
           _formKey.currentState?.validate();
         });
       }
+      ctrl.selection = TextSelection.collapsed(offset: ctrl.text.length);
```

Comment:
@wanlok what do you think of this simple solution?
I was thinking we need to find the root cause of the bounce first.

Reply:
I tried to comment out the validator logic, but the problem still persist.

Reply:
I think the next thing to look at is the ref.watch logic. See if a ref.watch in parent is causing the problem.

Reply:
I think we need to add more prints() to understand when the rebuild is happening.